code/dynamic_programming/src/egg_dropping_puzzle/EggDroppingPuzzle.java

```diff
@@ -0,0 +1,60 @@
+
+//Dynamic Programming solution for the Egg Dropping Puzzle
+ /* Returns the minimum number of attempts
+  needed in the worst case for a eggs and b floors.*/
+import java.util.*;
+public class EggDroppingPuzzle
+ 	{    

```

Comment:
Er, Indentation. Braces should be on the same level as their descriptors.

Reply:
Done